### PR TITLE
Enhancement: Improve layout of the BlockCapabilityBlockDocumentInput

### PR DESCRIPTION
**Pull request by pleek91**
# Description
Moves the logo inside the select
Exports the component

image

### CODE REVIEW

**Review by marichka-offen — APPROVED** [review]

File: src/components/BlockCapabilityBlockDocumentInput.vue [hunk 1]
```diff
@@ -1,16 +1,32 @@
 <template>
-  <div class="block-capability-block-document-input">
-    <BlockTypeLogo v-if="blockDocument" :block-type="blockDocument.blockType" />
-
-    <BlockDocumentsSelect v-model:selected="internalModalValue" class="block-capability-block-document-input__select" :block-documents="blockDocuments" required />
+  <div class="block-capability-block-document-input" :class="classes" :style="styles" v-bind="listeners">
```
Comment (znicholasbrown) [inline src/components/BlockCapabilityBlockDocumentInput.vue:2]:
```suggestion
  <div class="block-capability-block-document-input" :class="classes" :style="styles" v-bind="listeners">
```
Given that this is the root element I'm surprised to see this is necessary; is there something about the `useAttrsStylesClassesAndListeners` component that doesn't allow you just to use/bind the `attrs` down below?

Comment (pleek91) [reply src/components/BlockCapabilityBlockDocumentInput.vue:2]: We use this when we basically want to pass props/attrs but not styles, classes, or listeners. Specifically used here so `state` can be bound to the select.

Comment (znicholasbrown) [reply src/components/BlockCapabilityBlockDocumentInput.vue:2]: I missed an extremely critical line of your PR, sorry lol:
```
    inheritAttrs: false,
```

Comment (pleek91) [reply src/components/BlockCapabilityBlockDocumentInput.vue:2]: lol no worries!

**Pull request merged**